cli/src/commands/logs.ts

```diff
@@ -48,10 +55,27 @@ $ ${Command.cliBinName} logs --since 2m --follow`,
   public static strict = true;
 
   public async run() {
+    const { since, follow, limit } = this.parse(Logs).flags;
+    await this.authenticate();
 
-    const {flags: { since, follow, limit }} = this.parse(Logs);
-
-    this.log('Logs', { since, follow, limit });
+    const projectDir = await getProjectRootDir();
+    const projects = conf.get('projects') as Project[] | undefined || [];
+    const project = projects.find(({ directory }) => directory === projectDir);
+    if (project === undefined) {
+      return this.error(`No project deployments found, please run ${Command.cliBinName} deploy`);
+    }
+    let token;
+    do {
+      // TODO: support other envs
+      const sinceDate = typeof since === 'string' ? new Date(Date.now() - ms(since)) : since;
+      const { records, nextToken } = await this.lycanClient.getLogs(
+        project.applicationId.replace(/-/g, ''), project.defaultEnv, { follow, limit, since: sinceDate });
```

Comment:
Perhaps need to do it on the server, since this somewhat guarantees that `a-b-c` and `ab-c` are the same app...